### ISSUE TITLE
Avoid use of magic numbers

### ISSUE THREAD

**Issue by InfiniteCoder** [geeksocket/ecb-des#3]
magic numbers have been used in the code, [they should be avoided](https://stackoverflow.com/a/47902).

**Comment by bhavin192** [comment]
Yes, they should be avoided. I will be re factoring the whole code soon, where I will be focusing on readability of the code.
